aspnetcore/fundamentals/servers/kestrel/endpoints.md

```diff
@@ -5,60 +5,108 @@ description: Learn about configuring endpoints with Kestrel, the cross-platform
 monikerRange: '>= aspnetcore-5.0'
 ms.author: riande
 ms.custom: mvc
-ms.date: 03/10/2023
+ms.date: 06/14/2023
 uid: fundamentals/servers/kestrel/endpoints
 ---
 
 # Configure endpoints for the ASP.NET Core Kestrel web server
 
-[!INCLUDE[](~/includes/not-latest-version.md)]
+:::moniker range="< aspnetcore-7.0"
+[!INCLUDE [](~/includes/not-latest-version.md)]
+:::moniker-end
 
 :::moniker range=">= aspnetcore-8.0"
 
+An endpoint in Kestrel is defined by a combination of address and protocol. The address specifies the network interface and port on which the server listens for incoming requests. This can be a specific IP address, such as localhost or a public IP, along with a port number. The protocol defines the communication standard that is used between the client and server, such as HTTP or HTTPS. Endpoints can be configured to support HTTP or HTTPS or both, depending on the environment (development or production) and the requirements of the application. Endpoints in ASP.NET Core Kestrel provide the infrastructure for listening to incoming requests and routing them to the appropriate middleware.
```

Comment:
```suggestion
Endpoints in ASP.NET Core Kestrel provide the infrastructure for listening to incoming requests and routing them to the appropriate middleware. An endpoint in Kestrel is defined by a combination of address and protocol.

* The address specifies the network interface and port on which the server listens for incoming requests. This can be a specific IP address, such as localhost or a public IP, along with a port number.
* The protocol defines the communication standard that is used between the client and server, such as HTTP or HTTPS, or the HTTP version. Endpoints can be configured to support HTTP or HTTPS or both, depending on the environment (development or production) and the requirements of the application.
```
I think this is a bit better. It describes what an endpoint is used for first, then goes into detail about what an endpoint is.